docs/docs_for_team/README.md

```diff
@@ -128,14 +128,10 @@ agent {
 ### 5. Create the Jenkins job
 
 The last step is to create a new job that will use the Jenkinsfile of the repository.
-From Jenkins, 'New item' enter a name and select 'Pipeline'. There are only a few things that need to be changed.
-* Tick the 1st box that says "Github project" and enter the project url in the box that appears
-* In the pipeline section select "Pipeline script from SCM Pipeline"
-* Change the SCM to "git"
-* In the box that appears enter the project url in the box labeled "Repository URL"
-* If your project uses a file other than the Jenkinsfile then change the Script Path to reflect this
 
-This step is to test that everything is set up properly and that your job can actually do the actions defined in your jenkinsfile. Once this is done and your job works its creation can be automated.
+If you are writing a new groovy script then it may be useful to test it first using the Jenkins UI. This can be found under `Jenkins >> Manage Jenkins >> Script Console`.
+
+Currently our infrastructure only allows for one groovy file to be loaded to each Jenkins environment. Both the `complete_deployment_of_dns_and_jenkins` and the `gds_specific_dns_and_jenkins` examples have a custom script loaded at initialisation under `files >> custom-script.groovy`. In that file you need to edit the job block for your first job and add more job blocks as required in order that your jobs are loaded into Jenkins on initialisation.
```

Comment:
So this isn't the ADR, rather the docs on how to load up your own config. This file was already being run so we figured it wasn't too much of a stretch to use it as the baseboard from which to build config. 

We did look at adding in the extra script but thought that it would add complexity and possibly confuse users into thinking that creating jobs and setting up Jenkins need be separate things. Am happy to introduce this if you think it will help though.

Reply:
Thanks for correcting me on that, I was just pointing out something to consider, not intended to be a blocker.